install_config/router/f5_router.adoc

```diff
@@ -27,19 +27,41 @@ xref:../../install_config/router/default_haproxy_router.adoc#install-config-rout
 HAProxy router].
 
 [[install-router-f5-prerequisites]]
-== Prerequisites
+== Prerequisites and Supportability
 
 When deploying the F5 router plug-in, ensure you meet the following
 requirements:
 
-. An F5 host IP and credentials
-. The name of the virtual servers (for both HTTP and HTTPS)
-. The private key to the F5 instance
-. The host-internal IP and VXLAN gateway
+* A F5 host IP with:
+** Credentials for API access
+** ssh access via a private key
```

Comment:
it appears we use all caps for SSH when not used as a command:
s/ssh access/SSH access